Autopilot/Src/SensorFusion.cpp

```diff
@@ -86,4 +86,36 @@ SFError_t SF_GetAttitude(SFAttitudeOutput_t *Output, IMU_Data_t *imudata, Airspe
     return SFError;
 }
 
+void SF_Init() {
 
+}
+
+SFError_t SF_GenerateNewResult() {
+    SFError_t toReturn;
+    return toReturn;
+}
+
+SFError_t SF_GetResult(SFOutput_t *SFoutput) {
```

Comment:
Oh, you actually did create this. So you shouldn't have to comment out calls to sensor fusion in Path manager, right ?

Reply:
yeah true, I think I just forgot to uncomment those earlier parts haha